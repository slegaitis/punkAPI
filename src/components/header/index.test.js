import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './';

test('Header component should render', () => {
	const component = shallow(<Header />);
	const tree = toJson(component);

	expect(tree).toMatchSnapshot();
});
