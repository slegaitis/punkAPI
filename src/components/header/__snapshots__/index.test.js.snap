// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `
<header
  id="page-header"
>
  <div
    className="wrapper"
  >
    <h1>
      Glofox Test
    </h1>
  </div>
</header>
`;
